compiler/src/dotty/tools/dotc/parsing/Scanners.scala

```diff
@@ -233,6 +233,10 @@ object Scanners {
     /** Return a list of all the comments */
     def comments: List[Comment] = commentBuf.toList
 
+    /** Return a list of all the comment positions */
+    @deprecated("Use `comments` to get rich information source comments", since = "3.3.2")
```

Comment:
Why not mark it as `@InternalApi` as Akka?

Reply:
They can use ` comments.map(_.span)`.